code/items/equipments/TTTEquipment.cs

```diff
@@ -6,35 +6,36 @@
 
 namespace TTTReborn.Items
 {
-    [AttributeUsage(AttributeTargets.Class, Inherited = false)]
-    public class EquipmentAttribute : LibraryAttribute
-    {
-        public EquipmentAttribute(string name) : base(name)
-        {
+    // [AttributeUsage(AttributeTargets.Class, Inherited = false)]
+    // public class EquipmentAttribute : LibraryAttribute
+    // {
+    //     public EquipmentAttribute(string name) : base(name)
+    //     {
 
-        }
-    }
+    //     }
+    // }
```

Comment:
Can this just be removed?

Reply:
Kept it because it worked and afaik gonna work in the future again (still opened issue)